bridges/NordbayernBridge.php

```diff
@@ -39,14 +39,14 @@ class NordbayernBridge extends BridgeAbstract {
 				'Weißenburg' => 'weissenburg'
 			)
 		),
-		'policeReports' => array(
-			'name' => 'Police Reports',
-			'type' => 'checkbox',
-			'exampleValue' => 'checked',
-			'title' => 'Include Police Reports',
-		)
```

Comment:
It is still being used. In any case, removing this parameter will make feeds with "policeReports=1" invalid.